chia/_tests/cmds/wallet/test_did.py

```diff
@@ -3,25 +3,43 @@
 from pathlib import Path
 from typing import Optional, Union
 
+import pytest
 from chia_rs import G2Element
-from chia_rs.sized_bytes import bytes48
+from chia_rs.sized_bytes import bytes32, bytes48
 from chia_rs.sized_ints import uint32, uint64
 
 from chia._tests.cmds.cmd_test_utils import TestRpcClients, TestWalletRpcClient, logType, run_cli_command_and_assert
 from chia._tests.cmds.wallet.test_consts import FINGERPRINT_ARG, STD_TX, STD_UTX, get_bytes32
 from chia.rpc.wallet_request_types import DIDMessageSpendResponse, DIDTransferDIDResponse, DIDUpdateMetadataResponse
+from chia.types.blockchain_format.program import NIL, Program
 from chia.types.signing_mode import SigningMode
 from chia.util.bech32m import encode_puzzle_hash
 from chia.util.config import load_config
 from chia.wallet.conditions import Condition, ConditionValidTimes, CreateCoinAnnouncement, CreatePuzzleAnnouncement
+from chia.wallet.did_wallet.did_info import (
+    did_recovery_is_nil,
```

Comment:
interesting that ruff did this instead of deleting the trailing comma (at least that's what I assume happened)

Reply:
fixed